projects/igniteui-angular/src/lib/core/styles/components/card/_card-theme.scss

```diff
@@ -245,10 +287,16 @@
         flex: 1 1 auto;
         padding: $card-content-padding;
         color: --var($theme, 'content-text-color');
+
+        @if map-get($theme, variant) == 'bootstrap' {
+            margin-bottom: $card-content-margin;
+        }
     }
 
     %igx-card-content-text {
-        margin: $card-content-margin;
+        @if map-get($theme, variant) != 'bootstrap' {
```

Comment:
Again, it would be much easier to read the default value first, and remove the margin if the variant is 'bootstrap'.